ChaosMod/Effects/db/Player/PlayerLockCamera.cpp

```diff
@@ -1,23 +1,32 @@
 /*
-    Effect by dzwdz
+    Effect by dzwdz, modified
 */
 
 #include <stdafx.h>
 
+static void OnStop()
+{
+	ENABLE_CONTROL_ACTION(0, 1, true);
+	ENABLE_CONTROL_ACTION(0, 2, true);
+}
+
```

Comment:
I don't think that's necessary here? (since DISABLE_CONTROL_ACTION only applies to the current frame iirc)

Reply:
You’re correct